onnxruntime/python/tools/transformers/models/llama/convert_to_onnx.py

```diff
@@ -301,12 +515,48 @@ def get_args():
         help="Execution provider to verify parity with",
     )
 
+    parser.add_argument(
+        "-id",
+        "--device-id",
+        required=False,
+        type=str,
+        default="0",
+        help="Device ID for GPUs",
+    )
+
+    parser.add_argument(
+        "-r",
+        "--reexport",
+        required=False,
+        action="store_true",
+        help="Re-export models and overwrite existing models in output folder",
+    )
+    parser.set_defaults(reexport=False)
+
+    parser.add_argument(
+        "--no_merged",
+        required=False,
+        action="store_true",
+        help="Export models into 2 ONNX files instead of 1. Deprecated in favor of exporting into 1 ONNX file.",
+    )
+    parser.set_defaults(no_merged=False)
+
     parser.add_argument(
         "-q",
         "--quantization_method",
         default="",
-        choices=["smooth_quant", "quantize_dynamic"],
-        help="Run a specific quantization algorithm. Need to install extra packages in `requirements-quant.txt` for SmoothQuant.",
+        choices=["blockwise", "smooth_quant", "quantize_dynamic"],
+        help="Run a specific quantization algorithm (blockwise for int4, smooth_quant for int8, quantize_dynamic for int8). Blockwise is recommended. Need to install extra packages in `requirements-quant.txt` for SmoothQuant.",
+    )
+
+    blockwise_group = parser.add_argument_group("blockwise")
```

Comment:
nit: suggest to change "bitwise" to "4-bit quantization"

Reply:
The user has the option to specify `--precision int4` when exporting and then optionally provide the block size with `--block_size <int>` (default is 32). This option is only available for int4 quantization.

Reply:
When you run the script with --help, you will see that it is more clear that block_size is for 4-bit with proposed change.

Reply:
Fixed